external/common.cmake

```diff
@@ -0,0 +1,69 @@
+# -----------------------------------------------------------------------------
+# Numenta Platform for Intelligent Computing (NuPIC)
+# Copyright (C) 2016, Numenta, Inc.  Unless you have purchased from
+# Numenta, Inc. a separate commercial license for this software code, the
+# following terms and conditions apply:
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU Affero Public License for more details.
+#
+# You should have received a copy of the GNU Affero Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# -----------------------------------------------------------------------------
+#
+# This creates a library from external source code
+# that is physically included in this repository.
+#
+# Directions for adding a package:
+# 1) If it is an include only package, then just add it to external/include and skip the rest of these directions.
+# 2) Create a folder for the package in external/common/ and copy all files there.
+# 3) Add the .cpp files to the common_src list.
+# 4) The include files will be accessable with a path starting with the name of the folder created for your package.
+# 5) The objects will be placed in common.lib and included as part of the external libraries.
+#
+# Notes about MurmurHash3
```

Comment:
nit, I'd add this text to murmur hpp file, not here

Reply:
>  add this text to murmur hpp file,

It is in both places.

Reply:
> add this text to murmur hpp file,

It is in both places. But that might not be a good thing.  Someone might update the text in one place but not know about the other.   Removing it from common.cmake, leaving it only in MurmurHash3.hpp.